example/actions/image.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
-
```

Comment:
sorry, forgot to add the back. I had some linting issues at an earlier stage. Will add them back again.

Reply:
By the way, is 'use strict'; helping us here. Linter is complaining that it is unnecessary inside of modules. And since I have autofix on save on in VSCode, it just gets removed. I normally don't use strict mode in my own projects. Just want to know why you want it there.

Reply:
Hmm... won't this require node version > 14? I'm just afraid people will get errors in their projects. One thing is that the latest release of Node is > 14. But they still recommend 12.x.x. 
This is not an issue for someone messing around in a private project, but when you work for a company, you are sometimes not allowed to use the latest version. But I guess, you have already made Fluture dependent on version 14 now, right? So I guess it doesn't matter...

I will let this one be your call! :)

Reply:
> already made Fluture dependent on version 14

Not *dependent on*, but *compatible with*. Older versions of Node use the commonjs files by default.

> I will let this one be your call! :)

Sure. If that's okay I will make the desired changes and push them to your branch before merging.